.pre-commit-config.yaml

```
@@ -0,0 +1,21 @@
+# These hooks are meant to be triggered in CI to test the pre-commit hooks,
+# so rather than triggering at every commit, including locally, they are triggered
+# whenever the test-hooks CI workflow is run.
+default_stages: [manual]
+
+repos:
+  - repo: https://github.com/semgrep/pre-commit
+    rev: 'v1.76.0'
```

```suggestion
  - repo: local
```
Can we just make this a local hook so it is testing the current configuration, rather than being a pointer to potentially something else?

See https://pre-commit.com/#repository-local-hooks.

---

Looks like I can't just swap in `local` and there's more config fields that I need to understand. I'm gonna give up on trying that for now.

---

It may not be supported for the Python ones. Not sure.